src/server/apollo.context.spec.ts

```diff
@@ -452,284 +517,173 @@ describe('server/apollo.context', () => {
-            {
-              kind: 'FragmentDefinition',
-              name: {
-                value: 'UserFragment',
-              },
-            },
-            {
-              operation: 'query',
-              name: {
-                value: "GetMeApolloContext"
-              },
-              selectionSet: {
-                selections: [
-                  {
-                    name: {
-                      // the { query }
-                      value: 'me',
-                    },
-                    arguments: [],
-                    selectionSet: {
-                      selections: [
-                        {
-                          name: {
-                            value: 'UserFragment',
-                          },
-                        }
-                      ]
-                    }
-                  },
-                ],
-              },
-              variableDefinitions: [],
+    it('throw error when auth token & user jwt mismatch', () => {
+      context = new Context({
+        tokenConfig: JWTSIGNING_TOKEN_CONFIG,
+        token: AUTH_TOKEN
+      });
+      let telemetryMock: SinonMock = sandbox.mock(context.telemetry);
+      telemetryMock
+        .expects('error')
+        .once()
+        .withArgs(
+          sinonMatcher.string,
+          sinonMatcher({
+            error: {
+              message: 'Auth token and user jwt mismatch'
             },
-          ],
-        });
-
-        return true; // yep, that's what we're looking for
-      })
-      .reply(200, {
-        data: {
-          me: ME_FRAGMENT,
-        },
-      });
-
-      const meFragment = await context.fetchIdentityUser();
-      expect(meFragment).toMatchObject(ME_FRAGMENT);
-    });
-
-    it('does nothing when there is no data', async () => {
-      nock(IDENTITY_URL)
-      .post('/', (body: any) => body.query.match(/GetMeApolloContext/))
-      .reply(200, {});
-
-      const meFragment = await context.fetchIdentityUser();
-      expect(meFragment).toBeUndefined();
-    });
-
-    it('does nothing upon Error', async () => {
-      nock(IDENTITY_URL)
-      .post('/', (body: any) => body.query.match(/GetMeApolloContext/))
-      .reply(401);
-
-      const meFragment = await context.fetchIdentityUser();
-      expect(meFragment).toBeUndefined();
-    });
-
-    it('allows for overrides', async () => {
-      nock(IDENTITY_URL, {
-        reqheaders: {
-          authorization: OTHER_TOKEN,
-          [ TELEMETRY_HEADER_REQUEST_ID ]: 'REQUEST_ID',
-          'x-joy-custom': 'CUSTOM_HEADER',
-        },
-      })
-      .post('/', (body: any) => body.query.match(/GetMeApolloContext/))
-      .reply(200, {
-        data: {
-          me: ME_FRAGMENT,
-        },
-      });
-
-      const meFragment = await context.fetchIdentityUser({
-        token: OTHER_TOKEN,
-        headers: { 'x-joy-custom': 'CUSTOM_HEADER' },
-      });
-      expect(meFragment).toEqual(ME_FRAGMENT);
+            source: 'context#me',
```

Comment:
you can see pattern this in the rest of the file

```suggestion
            source: 'apollo',
```